statements-control/Library/DAOs/TypesDAO.cs

```diff
@@ -25,6 +26,7 @@ public override SqlParameter[] CreateParameters(MasterVO vo)
         {
             SqlParameter[] parameters = {
                 new SqlParameter("id",(vo as TypesVO).Id),
+                new SqlParameter("id",(vo as TypesVO).Name),
```

Comment:
Bom dia @thalees, poderia confirmar essa linha? 
Está escrita como "id" e pegando o valor "name" do objeto.
```suggestion
                new SqlParameter("Name",(vo as TypesVO).Name),
```

Reply:
Esta errado mesmo, fiz correndo e deixei passar isso ai